obspy/io/nordic/core.py

```diff
@@ -1981,56 +2091,64 @@ def nordpick(event, high_accuracy=True, nordic_format='OLD'):
             if arrival.time_weight is not None:
                 finalweight = _str_conv(int(round(
                     arrival.time_weight * 10))).rjust(2)[0:2]
-            if azimuth != ' ':
+            if backazimuth != ' ':
```

Comment:
if in doubt and some of these placeholders are set to '  ' (2 spaces) you could be a bit safer with a test like `if backazimuth.strip():`